src/libs/actions/OnyxDerived/configs/reportAttributes.ts

```diff
@@ -86,30 +86,33 @@ export default createOnyxDerivedValueConfig({
 
         if (isFullyComputed) {
             // if there are report-related updates, iterate over the updates
-            if (updates.length > 0) {
-                dataToIterate = prepareReportKeys(updates);
-            } else if (!!transactionsUpdates || !!transactionViolationsUpdates) {
-                let transactionReportIDs: string[] = [];
-                if (transactionsUpdates) {
-                    transactionReportIDs = Object.values(transactionsUpdates).map((transaction) => `${ONYXKEYS.COLLECTION.REPORT}${transaction?.reportID}`);
+            if (updates.length > 0 || !!transactionsUpdates || !!transactionViolationsUpdates) {
```

Comment:
NAB: Please convert to an early return (if the opposite of the condition is true, return like in the else block), and where possible below, like line 93. You can do this in a follow up.